website/content/api-docs/secret/databases/mysql-maria.mdx

```diff
@@ -46,16 +46,24 @@ has a number of parameters to further configure a connection.
 
 - `password` `(string: "")` - The root credential password used in the connection URL.
 
+- `auth_type` `(string: "")` - If set to `gcp_iam`, will enable IAM authentication to a Google
+  CloudSQL instance. For more information on authenticating to CloudSQL via IAM, please refer to
+  Google's official documentation [here.](https://cloud.google.com/sql/docs/postgres/authentication).
+  Requires Vault 1.15+.
```

Comment:
I think we can omit this type of Vault version information now that we have versioned docs?

Reply:
Ah cool good point!